person-service/src/main/kotlin/com/thehuginn/service/impl/DefaultAddressService.kt

```diff
@@ -0,0 +1,21 @@
+package com.thehuginn.service.impl
+
+import com.thehuginn.repository.AddressRepository
+import com.thehuginn.service.AddressService
+import com.thehuginn.service.result.mapper.AddressResultMapper
+import jakarta.enterprise.context.ApplicationScoped
+import jakarta.transaction.Transactional
+import java.util.*
```

Comment:
nepouzivat .* importy. Da sa to nestavit tusim Editor -> Code Style -> Kotlin -> Imports a nastav static import count na 999 napr